javascript/javascript1/week2/weatherwear.js

```diff
@@ -0,0 +1,7 @@
+function getOutfit  (temp){
+    let outfit = "";
+    if (temp < 10 || temp === 10 ) {return outfit = "Better  get in your boots and jacket !"}
```

Comment:
In these lines, you return an assignment to `outfit`. Can you explain why this works? 🤔 (It is a bit weird so it is fine if you cannot!)

In this case, you could return the string directly instead and avoid the variable `outfit` altogether. Can you spot other places in your homework for this week where you can remove a variable? 😄